NameLayerMain/src/vg/civcraft/mc/namelayer/database/AssociationList.java

```diff
@@ -136,13 +107,34 @@ public String getCurrentName(UUID uuid){
 	public void addPlayer(String playername, UUID uuid){
 		NameLayerPlugin.reconnectAndReintializeStatements();
 		PreparedStatement addPlayer = db.prepareStatement(this.addPlayer);
-		try {
-			addPlayer.setString(1, playername);
-			addPlayer.setString(2, uuid.toString());
-			addPlayer.execute();
-		} catch (SQLException e) {
-			// TODO Auto-generated catch block
-			e.printStackTrace();
+		String initalName = playername;
+		boolean insertedName = false;
+		for(int i = 0; !insertedName; i++) {
+			while(i != 0 && (playername.length() + String.valueOf(i).length()) > MAXIMUM_PLAYERNAME_LENGTH) { //ensure proper length
```

Comment:
Shorter != Easier to read, especially if instead of a some short lines, it's a few long ones

Anyway, it's pointless to argue over this, I'll change it w/e.